_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,288 @@
+
+## Background
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `p` element has a `line-height` of `20px` (equals `1.25em` ) which is below the permissible minimum, given the default pixel size of the body is 16 pixels. Since the [important flag][] is not set, the user can increase the `line-height` above the permissible minimum value.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+	<body>
+		<p style="line-height: 20px;">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</p>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+This `div` element has a `letter-spacing` of `0.12em` which is equal to the permissible minimum, given the default pixel size of the body is 16 pixels. Since the [important flag][] is not set, the user can increase the `letter-spacing` above the permissible minimum value.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+	<body>
+		<div style="letter-spacing: 0.12em;">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</div>
+	</body>
+</html>
+```
+
+#### Passed Example 3
+
+This `strong` element has a `word-spacing` of `1.92pt !important` (equals `0.12em` ) which is equal to the permissible value, given the default pixel size of the body is 16 pixels.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+	<body>
+		<strong style="word-spacing: 1.92pt !important;">
+			The boy walked down the street in a carefree way, playing without notice of what was about him. He didn't hear the
+			sound of the car as his ball careened into the road. He took a step toward it, and in doing so sealed his fate.
+		</strong>
+	</body>
+</html>
+```
+
+#### Passed Example 4
+
+This `article` element has both `word-spacing` and `line-height` specified, which are both below the permissible minimum, given the default pixel size of the body is 16 pixels. Since the [important flag][] is not set for either the `line-height` or `word-spacing`, the user can increase them to be above the permissible minimum value.
```

Comment:
```suggestion
This `article` element has both `word-spacing` and `line-height` specified, which are both below the permissible minimum, given the pixel size of the body is 16 pixels. Since the [important flag][] is not set for either the `line-height` or `word-spacing`, the user can increase them to be above the permissible minimum value.
```